src/layout/Cards/CardContext.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import type { PropsWithChildren } from 'react';
+
+import { createContext } from 'src/core/contexts/context';
+import type { LayoutNode } from 'src/utils/layout/LayoutNode';
+
+interface CardContext {
+  node: LayoutNode<'Cards'>;
+  renderedInMedia: boolean;
+  minMediaHeight?: string;
+}
+
+const { Provider, useCtx } = createContext<CardContext | undefined>({
+  name: 'Card',
+  required: false,
+  default: undefined,
+});
+
+export function CardProvider({ children, ...state }: PropsWithChildren<CardContext>) {
```

Comment:
Meg og min masing om contexter igjen 🙈 
Går det an å heller bruke en funksjon ala dette i stedet for contexten som `Video`, `Audio` og `ImageComponent` bruker? Da kan man lettere endre på hver enkelt om behovene endres 😊 
```typescript
export function getCardsConfig(node: LayoutNode, defaultHeight = '150px') {
  const cardsParent = findParentNodeOfType<LayoutNode<'Cards'>>(node, 'Cards');
  const cardMediaHeight = parseSize(cardsParent?.item.minMediaHeight, defaultHeight);

  return { hasCardsParent: !!cardsParent, cardMediaHeight };
}
```